__tests__/praxis/arthur-deviation-weights/unit.ts

```diff
@@ -30,22 +29,26 @@ describe('ArthurDeviationWeights Class: Unit', () => {
       const inputDeltas = [[1]];
       const width = 3;
       const height = 1;
-      const kernel = gpuMock(update, {
+      const kernel: any = gpuMock(update, {
```

Comment:
`gpuMock` does specify return type `IKernelRunShortcut`

Reply:
for this, if we specify IKernelRunShortcut type here then the result tyoe in next line will be an because of the return type of kernel and input type of shave function...